orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -162,7 +163,9 @@ class ManualJudgmentStage extends LinearStage implements RestartableStage, Authe
           application: stage.execution.application
         ),
         additionalContext: [
-          instructions: stage.context.instructions ?: ""
+          instructions: ContextParameterProcessor.process([
```

Comment:
@tomaslin - Shouldn't the processing happen automatically without the need to explicitly process it?